docs/readthedocs_theme/base.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   {% block htmltitle %}
   <title>{{ page_title }}</title>
+  {% if page_description %}<meta name="description" content="{{ page_description }}">{% endif %}
```

Comment:
Although obviously, sequelizejs.com should be the primary source of SEO stuff, but might as well have the basics on the docs so they are easy to find :)